api/hooks.go

```diff
@@ -97,14 +101,18 @@ func (w *Webhook) trigger() error {
 		switch rsp.StatusCode {
 		case http.StatusOK, http.StatusNoContent, http.StatusAccepted:
 			rspLog.Infof("Finished processing webhook in %s", dur)
-			return nil
+			var body io.ReadCloser
+			if rsp.ContentLength > 0 {
+				body = rsp.Body
+			}
+			return body, nil
```

Comment:
I think that instead of returning a possibly nil value here we should return something like `http.NoBody` so that you only have to check for an error as the consumer